lib/logmsg/tests/test_nvtable.c

```diff
@@ -54,77 +52,6 @@
     }                                                                   \
   while (0)
 
-static void
-test_nv_registry()
-{
-  NVRegistry *reg;
-  gint i, j;
-  NVHandle handle, prev_handle;
-  const gchar *name;
-  gssize len;
-  const gchar *builtins[] = { "BUILTIN1", "BUILTIN2", "BUILTIN3", NULL };
-
-  reg = nv_registry_new(builtins);
-
-  for (i = 0; builtins[i]; i++)
-    {
-      handle = nv_registry_alloc_handle(reg, builtins[i]);
-      TEST_ASSERT(handle == (i+1));
-      name = nv_registry_get_handle_name(reg, handle, &len);
-      TEST_ASSERT(strcmp(name, builtins[i]) == 0);
-      TEST_ASSERT(strlen(name) == len);
-    }
```

Comment:
As I see, `nv_registry_get_handle_name` is tested only in this testcase, so we should keep this part .